x/oracle/client/cli/tx.go

```diff
@@ -13,20 +13,26 @@ import (
 	"github.com/spf13/cobra"
 	tmtime "github.com/tendermint/tendermint/types/time"
 
+	dnTypes "github.com/dfinance/dnode/helpers/types"
 	"github.com/dfinance/dnode/x/oracle/internal/types"
 )
 
 // GetCmdPostPrice cli command for posting prices.
 func GetCmdPostPrice(cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "postprice [from_key_or_address] [assetCode] [price] [receivedAt]",
+		Use:   "postprice [fromKeyOrAddress] [assetCode] [price] [receivedAt]",
 		Short: "post the latest price for a particular asset",
 		Args:  cobra.ExactArgs(4),
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Relative to all genesis, query, tx cmds: we should use helpers.BuildCmdHelp for "out of the box" documentation